pkgs/tools/system/nvidia-system-monitor-qt/default.nix

```diff
@@ -0,0 +1,66 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, cmake
+, qtbase
+, wrapQtAppsHook
+, makeDesktopItem
+, copyDesktopItems
+}:
+
+let
+  # Based in desktop files from official packages:
+  # https://github.com/congard/nvidia-system-monitor-qt/tree/master/package
+  desktopItem = makeDesktopItem {
+    name = "nvidia-system-monitor-qt";
+    desktopName = "NVIDIA System Monitor";
+    icon = "qnvsm";
+    exec = "qnvsm";
+    categories = [
+      "System"
+      "Utility"
+      "Qt"
+    ];
+  };
```

Comment:
We can also just copy the desktop file

Reply:
None of them are quite right for Nix. The DEB and RPM versions hardcode the binary path, and the Flatpak uses a different icon name.

I figured it would be simpler to recreate the file than worry about patching things like these.

Reply:
We can just patch the binary path and icon name instead of copying everything

Reply:
We don't have a clear guideline on that. That preference is up to the package author.